ci/cron/monthly.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Azure Pipelines file, see https://aka.ms/yaml
+
+# Do not run on PRs
+pr: none
+
+# Do not run on merge to master
+trigger: none
+
+# Run on schedule: first Monday of each month at 4AM UTC
+schedules:
+  - cron: '0 4 1-7 * Mon'
```

Comment:
Is there a specific reason why we send it on the first Monday instead of just sending it on the 1st which seems a bit simpler?

Reply:
That's the day Security wants to receive it.